src/components/List/List.tsx

```diff
@@ -65,12 +65,15 @@ class List extends UIComponent<any, any> {
   static itemProps = ['debug', 'selection', 'truncateContent', 'truncateHeader', 'variables']
 
   renderComponent({ ElementType, classes, rest }) {
-    const { items } = this.props
+    const { items, children } = this.props
     const itemProps = _.pick(this.props, List.itemProps)
+    const shorthandContent = _.map(items, item =>
+      ListItem.create(item, { defaultProps: itemProps }),
+    )
```

Comment:
If `children` are provided, then we do not need to loop over items and create them.  Suggest pulling this `_.map` into a class method called `renderItems = () => { ... }`, then using it like so:

```jsx
<ElementType {...rest} className={classes.root}>
  {childrenExist(children) ? children : this.renderItems()}
</ElementType>
```

This way, we only perform this operation when necessary.